knox_project/settings.py

```diff
@@ -59,5 +59,6 @@
 TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
 
 REST_KNOX = {
-    'AUTO_REFRESH': True
+    'AUTO_REFRESH': True,
+    'TOKEN_LIMIT': None
```

Comment:
Again I had set a bad example for this. But lets avoid having settings here as #111 has made me aware it is cleaner without.